default.nix

```diff
@@ -75,18 +75,25 @@ stdenv.mkDerivation rec {
         (path: _:
            !elem (baseNameOf path) [".git" "result" "bin" "_build" "_build_ci"]) ./.;
 
+  preConfigure = "patchShebangs kernel/";
+
   prefixKey = "-prefix ";
 
   buildFlags = [ "world" "byte" ] ++ optional buildDoc "doc-html";
 
   installTargets =
     [ "install" "install-byte" ] ++ optional buildDoc "install-doc-html";
 
+  createFindlibDestdir = !shell;
```

Comment:
What would this do if you simply put `true` instead?

Reply:
Running `nix-shell` would then fail because it cannot create a directory in your store. (I did try and got bitten.)

Reply:
Isn't it a bug of nixpkgs' OCaml package or something like this?